src/MovieList.tsx

```diff
@@ -0,0 +1,27 @@
+import './MovieList.css'
+import React from 'react'
+
+export interface JSONMovie {
+  Poster: string
```

Comment:
You should use JavaScript naming convention.
https://google.github.io/styleguide/jsguide.html#naming-parameter-names